appveyor.yml

```diff
@@ -27,6 +27,14 @@ install:
   - set GCC_URL=http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-gcc
   - set GCC_EXT=5.4.0-1-any.pkg.tar.xz
   - bash -lc "pacman -U --noconfirm $GCC_URL-$GCC_EXT $GCC_URL-ada-$GCC_EXT $GCC_URL-fortran-$GCC_EXT $GCC_URL-libgfortran-$GCC_EXT $GCC_URL-libs-$GCC_EXT $GCC_URL-objc-$GCC_EXT"
+  # ffmpeg install
+  - SET PACKAGE_URL=http://www.guillaume-gomez.fr/packages/win64.zip
```

Comment:
Do we really need this package? I'm not super excited about a binary blob hosted on your personal server, but if we need it, we can instead host it on one of the Servo s3 accounts.